src/Checkbox.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Icon from './Icon'
+import Flex from './Flex'
+import Box from './Box'
+import theme from './theme'
+
+class Checkbox extends React.Component {
+  constructor({ value, isChecked }) {
+    super()
+    this.state = {
+      value,
+      isChecked
```

Comment:
By using React's form props, you can make this component completely stateless, which means it could be used as either a controlled or uncontrolled form element